src/cript/api/api.py

```diff
@@ -27,6 +27,8 @@
 # Use the `_get_global_cached_api for access.
 _global_cached_api = None
 
+_API_VERSION_STRING = "v1"
```

Comment:
I suggest putting this within `_prepare_host()` function to encapsulate it in there because it does not need to be available for the whole class. The only function that should mess with it is `_prepare_host`

Reply:
Yes, agreed if that was true.
But for the construction of node URLs we also need the API_VERSION_STRING.
See primary_base_node:44.

Ideally this would be supplied by the connection to the API at the hand shake, but until then, I wanted it in a central location, where it can be accessed from everywhere we need it.

Reply:
oh I see, it seems like we are reconstructing the host while we are making the UUID, I think a better idea would be if we have the host cached somewhere and after the API is set every place gets it directly from the host and doesn't have to change anything because if tomorrow the host changes from `host/api/v1` to `host/my_api/v2` then we would just change the way the host is constructed in one file and all other files that are using it would fall into place correctly

Reply:
That is possible too.
It would be a api function somehow.
And hopefully in the future the API comminucates this on hand shake.
Do you want to open a low priority trello ticket for this?

In the mean time, I can implement this as an API member function, to get host/api/version from the cached API.

Reply:
I'm thinking since we have a cached API the other functions can just get the host from the api property?
what do you think? does that work/

Reply:
I can add this in pretty quickly right now as well

Reply:
Just pushed that